scripts.py

```diff
@@ -539,6 +539,8 @@ def refresh_metadata(self, identifier):
             provider = OverdriveBibliographicCoverageProvider
         elif identifier.type==Identifier.AXIS_360_ID:
             provider = Axis360BibliographicCoverageProvider
+        elif identifier.type==Identifier.ENKI_ID:
+            provider = EnkiBibliographicCoverageProvider
```

Comment:
Right. We just thought the Axis import was missing but I'm guessing it's not required. And I refactored the EnkiBibliographicCoverageProvider out of core so I'll delete that.